Dockerfile

```diff
@@ -1,5 +1,13 @@
+FROM maven:3.5-jdk-8-alpine AS build
+COPY pom.xml /work/pom.xml
+WORKDIR /work
+RUN mvn dependency:go-offline
+COPY . /work
+RUN mvn clean install
+
+# --- to be distributed as jenkins/jenkins:lts-alpine-casc
 FROM jenkins/jenkins:lts-alpine
 
-COPY --chown=jenkins target/configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.jpi
+COPY --chown=jenkins --from=build /work/target/configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.jpi
```

Comment:
it is missing all the configuration-as-code dependencies

Reply:
sole dependency is job-dsl (and it's dependency) which we should consider optional (need to update configurator accordingly)